meta/main.yml

```
@@ -0,0 +1,22 @@
+---
+galaxy_info:
+  author: haslersn
+  description: Network configuration using systemd.network; see https://www.freedesktop.org/software/systemd/man/systemd.network.html
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: AGPLv3
+
+  min_ansible_version: 2.4
```

```suggestion
  min_ansible_version: 2.6
```
See https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#dict-filter